test/ask-permission.js

```diff
@@ -0,0 +1,58 @@
+import {spawn} from 'child_process';
+import test from 'ava';
+
+test.cb('skip in TTY mode', t => {
+	t.plan(1);
+
+	const insProcess = spawn('node', [
```

Comment:
I think you can use [`execa`](https://github.com/sindresorhus/execa) instead.

```
test('skip in TTY mode', async t => {
	const {code} = await execa('node', [./test/fixtures/sub-process.js]);
	t.is(code, 145);
});
```

Do it on the the following tests too.